lib/ansible/modules/network/meraki/meraki_device.py

```diff
@@ -272,7 +272,7 @@ def main():
     meraki.params['follow_redirects'] = 'all'
 
     query_urls = {'device': '/networks/{net_id}/devices'}
-    query_org_urls = {'device': '/organizations/{org_id}/deviceStatuses'}
+    query_org_urls = {'device': '/organizations/{org_id}/inventory'}
```

Comment:
inventory ?

Reply:
Yeah. Slightly different endpoint. I'm not clear what you're asking though.

Reply:
It looked weird, but if this is as intended, no issues here.